pytorch/docker-compose.yaml

```diff
@@ -71,6 +71,8 @@ services:
       - 8888:8888
   multinode:
     build:
+      args:
+        SSH_KEY_PATH: ${SSH_KEY_PATH}
```

Comment:
If you specify an arg, make sure it has a default value.

Reply:
addressed accordingly